java-squid/src/main/java/org/sonar/java/ast/parser/ListTreeImpl.java

```diff
@@ -60,14 +60,17 @@ public boolean isLeaf() {
 
   @Override
   public void accept(TreeVisitor visitor) {
+    for (T t : list) {
+      ((Tree) t).accept(visitor);
+    }
     // TODO
-    throw new UnsupportedOperationException("On class: " + getClass().getSimpleName());
+//    throw new UnsupportedOperationException("On class: " + getClass().getSimpleName());
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) This block of commented-out lines of code should be removed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3ACommentedOutCodeLine)